Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -67,13 +124,21 @@ class TabTrayViewController: UIViewController, TabTrayController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
         setupView()
+        listenForThemeChange(view)
+        updateToolbarItems()
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        updateToolbarItems()
     }
 
     func applyTheme() {
```

Comment:
nit; we could structure that VC with marks already (in other places too) since this will grow
```suggestion
    // MARK: Themeable

    func applyTheme() {
```